tests/test_trackingstore.py

```diff
@@ -330,6 +330,64 @@ def test_create_run(mocker, mlflow_parent_run_id, faculty_parent_run_id):
     assert returned_run == mlflow_run
 
 
+def test_create_run_experiment_not_active_conflict(mocker):
+    mlflow_timestamp = mocker.Mock()
+    faculty_datetime = mocker.Mock()
+    timestamp_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore."
+        "mlflow_timestamp_to_datetime_milliseconds",
+        return_value=faculty_datetime,
+    )
+
+    mlflow_tag = mocker.Mock()
+    faculty_tag = mocker.Mock()
+    tag_converter_mock = mocker.patch(
+        "mlflow_faculty.trackingstore.mlflow_tag_to_faculty_tag",
+        return_value=faculty_tag,
+    )
+
+    mock_client = mocker.Mock()
+    exception = faculty.clients.experiment.ExperimentNotActiveConflict(
+        message="message", experiment_id="test-id"
+    )
+
+    mock_client.create_run.side_effect = exception
+    mocker.patch("faculty.client", return_value=mock_client)
+    experiment_id = mocker.Mock()
+    run_name = mocker.Mock()
+
+    store = FacultyRestStore(STORE_URI)
+
+    with pytest.raises(MlflowException, match="experiment"):
+        with pytest.raises(
```

Comment:
This inner `pytest.raises` is not correct. The test passes for reasons I'd rather explain in person.